src/olympia/discovery/management/commands/extract_content_strings.py

```diff
@@ -72,6 +72,12 @@ class DiscoItemAPIParser(BaseAPIParser):
     fields = ('custom_heading', 'custom_description')
 
 
+class PrimaryHeroShelfAPIParser(BaseAPIParser):
+    api = settings.PRIMARY_HERO_EDITORIAL_CONTENT_API
+    l10n_comment = 'editorial content for the primary hero shelves.'
+    fields = ('description',)
```

Comment:
actually, we want to be translating the raw description field on the model... mmm.
- a special view&serializer just for exposing description (like DiscoItemAPIParser)?
- a `?raw` parameter on https://addons-server.readthedocs.io/en/latest/topics/api/hero.html#get--api-v4-hero-primary- ?
- return an extra `custom_description` or `description_raw` field if `?all=True`?
- something else?

Reply:
I'd go with the extra special view I think, just to make it consistent with the way we do it for discovery items, but I don't care about this very strongly...

Reply:
added the ?raw param as it seemed the least disruptive.